lib/ansible/modules/packaging/language/maven_artifact.py

```diff
@@ -145,12 +145,21 @@
     group_id: org.springframework
     dest: /tmp/
     keep_name: yes
+
+# Download the latest version of the JUnit framework artifact from Maven local
+- maven_artifact:
+    group_id: junit
+    artifact_id: junit
+    dest: /tmp/junit-latest.jar
+    repository_url: "file://{{ lookup('env','HOME') }}/.m2/repository"
```

Comment:
It would be nice to also add something about the scheme in the `repo` parameter documentation.

Reply:
done